charts/kube-state-metrics-v2/Chart.yaml

```diff
@@ -1,13 +1,13 @@
 apiVersion: v1
-name: kube-state-metrics
+name: kube-state-metrics-v2
 description: Install kube-state-metrics to generate and expose cluster-level metrics
 keywords:
 - metric
 - monitoring
 - prometheus
 - kubernetes
-version: 2.13.0
-appVersion: 1.9.8
+version: 2.0.0
```

Comment:
We should have a target in the Makefile that non-helm users can execute to update these versions before release.